src/nameserver/system_table.h

```diff
@@ -64,6 +66,23 @@ class SystemTable {
                 ttl->set_lat_ttl(1);
                 break;
             }
+            case SystemTableType::KPreAggMetaInfo: {
+                SetColumnDesc("aggr_table", openmldb::type::DataType::kString, table_info->add_column_desc());
+                SetColumnDesc("base_db", openmldb::type::DataType::kString, table_info->add_column_desc());
+                SetColumnDesc("base_table", openmldb::type::DataType::kString, table_info->add_column_desc());
+                SetColumnDesc("aggr_func", openmldb::type::DataType::kString, table_info->add_column_desc());
+                SetColumnDesc("aggr_col", openmldb::type::DataType::kString, table_info->add_column_desc());
+                SetColumnDesc("partition_cols", openmldb::type::DataType::kString, table_info->add_column_desc());
+                SetColumnDesc("order_by_col", openmldb::type::DataType::kString, table_info->add_column_desc());
```

Comment:
also add an index for cols (base_db, base_table, partition_cols, order_by_col, aggr_func, aggr_col), which can uniquely identify an `aggr_table`

Reply:
We should also add `bucket size` to the index columns to uniquely identify a pre-aggregated table.